src/ensembl/src/shared/components/tabs/Tabs.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+import times from 'lodash/times';
+
+import Tabs, { Tab, TabsProps } from './Tabs';
+
+const onTabChange = jest.fn();
+
+const createTabGroup = (): Tab[] => {
+  const options = times(10, () => ({
+    title: faker.random.words(),
+    isDisabled: faker.random.boolean()
+  }));
+
+  // Make sure we have atleast one enabled and one disabled entry
+  options.push({
+    title: faker.random.words(),
+    isDisabled: true
+  });
+  options.push({
+    title: faker.random.words(),
+    isDisabled: false
+  });
+
+  return options;
+};
+
+const tabsData = createTabGroup();
+
+const tabClassNames = {
+  selected: faker.lorem.word(),
+  default: faker.lorem.word(),
+  disabled: faker.lorem.word(),
+  tabsContainer: faker.lorem.word()
+};
+
+const defaultProps = {
+  tabs: tabsData,
+  selectedTab: tabsData.find((tab) => !tab.isDisabled)?.title || '',
```

Comment:
The `|| ''` here is a little bit of a lie: `tabsData` guarantee that there is at least one disabled tab, so the find function will never return null. Would typescript be happy if you just indicated that the selected tab is always a string?

`selectedTab: tabsData.find((tab) => !tab.isDisabled)?.title as string` ?

or

`selectedTab: (tabsData.find(tab => !tab.isDisabled) as {title: string}).title` ?

Reply:
`(tabsData.find((tab) => !tab.isDisabled) as Tab).title` seems to work.